src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -30,7 +30,43 @@ void draw_settings_dialog(HostState &host) {
     ImGuiTabBarFlags settings_tab_flags = ImGuiTabBarFlags_None;
     ImGui::BeginTabBar("SettingsTabBar", settings_tab_flags);
 
+    // Core
+    if (ImGui::BeginTabItem("Core")) {
+        ImGui::PopStyleColor();
+        ImGui::Text("LLE Module \nSet your LLE Module Wanted.");
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Write exemple 'libpgf', press Apply and re write other.");
+        std::string lle_module;
+        ImGui::InputTextWithHint("Set lle module", "Add lle module wanted", &lle_module);
+        if (ImGui::Button("Apply add LLE Modules")) {
+            if (!lle_module.empty()) {
+                host.cfg.lle_modules.push_back(lle_module);
+            }
+        }
+        ImGui::SameLine();
+        if (ImGui::Button("Reset LLE Modules")) {
```

Comment:
if (ImGui::Button("Reset LLE Modules.")) {

Reply:
I have Nevers est.  Inside butin is wrong